Pakrun/settings.py

```diff
@@ -25,7 +25,7 @@
 SECRET_KEY = 'django-insecure-!x7)v_!)*c3$f)s-x867_#q74dvjq*(dwvh8rw*4gwcp6)$kw6'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False

```

Comment:
I think we still need this ,the app is still under development

Reply:
which one?

Reply:
i was testing few  things